code/modules/recycling/conveyor2.dm

```diff
@@ -9,7 +9,7 @@
 	layer = CONVEYOR_LAYER // so they appear under stuff
 	anchored = 1
 	var/operating = 0	// 1 if running forward, -1 if backwards, 0 if off
-	var/operable = 1	// true if can operate (no broken segments in this belt run)
+	var/is_operational = 1	// true if can operate (no broken segments in this belt run)
```

Comment:
```suggestion
	var/is_operational = TRUE	// true if can operate (no broken segments in this belt run)
```

Reply:
Oof that's one I missed in the mass replacement. I'll likely not change this value, let me see.